internal/controller/cephfscg/volumegroupsourcehandler.go

```diff
@@ -30,6 +30,8 @@ var (
 	RestorePVCinCGNameFormat      = "vs-cg-%s"
 	SnapshotGroup                 = "snapshot.storage.k8s.io"
 	SnapshotGroupKind             = "VolumeSnapshot"
+
+	createdByRamenLabel = "ramendr.openshift.io/resource-created-by-ramen"
```

Comment:
This is defined in multipe files.  Why repeating? Can't you just put it once in the `util` package?
It is used throughout, it should start with a capital C in this case.